### PR TITLE
Updated copyright notice

### PR DESCRIPTION
**Pull request by WSShawn**
Updated copyright notice to 2022

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 title: sklearn-evaluation
 author: Ploomber
-copyright: Ploomber
+copyright: 2022
```
Comment (idomic) [inline docs/_config.yml:3]: Same as last one, we're at 2023 already

Comment (idomic) [reply docs/_config.yml:3]: Make sure you patch both changes in a single PR.

Comment (idomic) [reply docs/_config.yml:3]: @WSShawn ^

**Comment by edublancas** [comment]
I see another PR was merged with the fixed value, closing this one

**Pull request closed without merging**